shipyard_blueprints/dbt/shipyard_dbt/dbt.py

```diff
@@ -1,14 +1,28 @@
+import os
+import platform
+
 import requests
-from shipyard_templates import Etl
+from shipyard_templates import Etl, ShipyardLogger, ExitCodeException
+
+logger = ShipyardLogger.get_logger()
 
 
 class DbtClient(Etl):
+    EXIT_CODE_FINAL_STATUS_SUCCESS = 0
+    EXIT_CODE_UNKNOWN_ERROR = 3
```

Comment:
Not a huge deal, but recently we have been putting the exit codes along with custom exceptions in another file